### DISCUSSION TITLE
An error occuring during the task

### DISCUSSION THREAD

**Discussion by JackKaif** [Equim-chan/mjai-reviewer#150] [Q&A] [answered]
Mahjong Soul, 4-player tonpuusen game, standard ruleset. Trying to use akochan (as Mortal works only with hanchan), but got an error. Site's page and error's screens are attached.

![image](https://github.com/user-attachments/assets/bc688d0e-e171-4f89-a98e-dc0fa407b236)

![image](https://github.com/user-attachments/assets/b7f519c8-a108-460b-bc60-7e576d87b8dd)
Game link: https://mahjongsoul.game.yo-star.com/?paipu=240914-e47f58e2-d245-47ed-97b5-1eed1fc100c1_a925013524

**Comment by Equim-chan** [comment, marked as answer]
Try again, it should be fixed now.

**Reply by morimori1014** [reply]
The same error is occurring.
